app/views/layouts/internal.html.erb

```diff
@@ -16,7 +16,8 @@
     <%= yield :stylesheets %>
   </head>
   <body>
-    <div id="app" class="h-screen">
+    <div id="app" class="h-screen relative">
```

Comment:
É necessário esse relative?
Eu fiz alguns testes aqui e me parece apresentar o mesmo resultado se puder dar um a conferida.

Reply:
Pelo que pesquisei, quando não setamos o relative em um elemento que seria o pai, ele posiciona o elemento com o relative em relação a tag body. No nosso caso não vai fazer diferença pq a posição do alert ficará no canto superior direito em relação a tela toda que é o body. Logo, não tem necessidade de colocar o relative nessa div. https://dev.to/domagojvidovic/position-absolute-and-relative-the-simplest-guide-5169#:~:text=Whenever%20you%20set%20position%3A%20absolute,relative%20to%20the%20body%20element.

Reply:
Boaa blz!